src/run.py

```diff
@@ -1,13 +1,13 @@
 # -*- coding: utf-8 -*-
-"""
-Presence analyzer web app.
-"""
+"""Presence analyzer web app."""
+
+# Standard libraries
 import os.path
 import logging.config
 
+# Local libraries
 from presence_analyzer.main import app
-import presence_analyzer.views
-
+# import presence_analyzer.views
```

Comment:
In this unique case, which is the `run.py` file, you must uncomment: `import presence_analyzer.views`
This import is necessary for the application to work.